2_process/src/fetch_func.R

```diff
@@ -0,0 +1,27 @@
+
+fetch_data <- function (download_dir) {
+  
+  library(dplyr)
+  library(readr)
+  library(stringr)
+  library(sbtools)
+  
+  # Get the data from ScienceBase
+  mendota_file <- file.path(download_dir, 'model_RMSEs.csv')
+  item_file_download('5d925066e4b0c4f70d0d0599', names = 'me_RMSE.csv', destinations = mendota_file, overwrite_file = TRUE)
+  
+  # Prepare the data for plotting
+  eval_data <- readr::read_csv(mendota_file, col_types = 'iccd') %>%
```

Comment:
We want to separate this function into a `fetch` step (which should go into a `1_fetch/src/` folder and a process step, which can live in `2_process/src`